core/domain/exp_services_test.py

```diff
@@ -1222,6 +1223,62 @@ def test_save_original_and_compressed_versions_of_image(self):
         self.assertEqual(
             fs.isfile('image/%s' % self.MICRO_IMAGE_FILENAME), True)
 
+    def test_save_original_and_compressed_versions_of_image_with_dev_mode_off(
+            self):
+        dev_mode_swap = self.swap(constants, 'DEV_MODE', False)
```

Comment:
nit: name this `prod_mode_swap` since that's what we're turning on.

Reply:
Done